src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4 {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5 {
+  background-image: url(../images/star.svg);
```

Comment:
Ensure that the paths to the images `star-active.svg` and `star.svg` are correct relative to the SCSS file location. Incorrect paths will result in missing images on the page.